From 46fad54f8706589be2ce0ef898731478f6baa54d Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Wed, 14 Dec 2016 13:10:41 +0000
Subject: [PATCH] source/simple: Greatly increase the default timeout

We've seen some failed downloads from the 15s timeout so hit it up to
2 minutes, "just in case".

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/builder/source/simple.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/builder/source/simple.go b/src/builder/source/simple.go
index c6e7d09..28a48d8 100644
--- a/src/builder/source/simple.go
+++ b/src/builder/source/simple.go
@@ -111,7 +111,7 @@ func (s *SimpleSource) IsFetched() bool {
 func (s *SimpleSource) download(destination string) error {
 	// Fix up the http client
 	client := http.Client{
-		Timeout: time.Second * 15,
+		Timeout: time.Minute * 2,
 	}
 	// Grab a http request
 	resp, err := client.Get(s.URI)
-- 
2.11.0

